internal/generate/serviceendpointtests/file.tmpl

```diff
@@ -0,0 +1,496 @@
+// Code generated by internal/generate/serviceendpointtests/main.go; DO NOT EDIT.
+
+package {{ .PackageName }}_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"maps"
```

Comment:
Until we are targeting Go 1.21 this will need to be "golang.org/x/exp/maps".

Reply:
8 August 2023 😂